db/migrate/20170310182013_add_fingerprint2_to_schemas.rb

```diff
@@ -0,0 +1,9 @@
+class AddFingerprint2ToSchemas < ActiveRecord::Migration[5.0]
+  def change
+    add_column(:schemas, :fingerprint2, :string, null: true)
+    add_index(:schemas, :fingerprint2, unique: true)
+
+    remove_index(:schemas, :fingerprint)
+    add_index(:schemas, :fingerprint, unique: false)
```

Comment:
I think it's fine to populate both. My question is really why are we dropping the "unique" part of the the `fingerprint` index?